<?xml version="1.0" encoding="utf-8"?>

<launch>

    <!-- Arguments -->
    <arg name="images_dir" default="/datasets/ODEMAR/130f/" />
    <arg name="working_dir" default="/home/emilio/Escritorio/Mosaicing/" />
    <arg name="img_descriptor" default="FAST_LDB" />
    <arg name="nkeypoints" default="3000" />    

    <!-- Mosaic Builder node -->
    <node pkg="bimos" type="bimos_node" name="bimos_node" output="screen">
        <param name="images_dir" value="$(arg images_dir)" />
        <param name="working_dir" value="$(arg working_dir)" />
        <param name="img_descriptor" value="$(arg img_descriptor)" />
        <param name="nkeypoints" value="$(arg nkeypoints)" />        
    </node>

</launch>
